doc/sql.extensions/README.sql_package.md

```diff
@@ -42,5 +43,21 @@ select *
   }');
 ```
 
+## Procedure `PARSE_UNQUALIFIED_NAMES`
+
+`RDB$SQL.PARSE_UNQUALIFIED_NAMES` is a selectable procedure that parses a list of unqualified SQL names and returns
+one row for each name. The input must follow parse rules for names and the output of unquoted names are uppercased.
+
+```sql
+select *
+  from rdb$sql.parse_unqualified_names('schema1, schema2, "schema3", "schema 4", "schema ""5"""');
+
+-- SCHEMA1
+-- SCHEMA2
+-- schema3
+-- "schema 4"
+-- "schema "5"
```

Comment:
-- "schema "5"" ? Looks like last double quote is missed

Reply:
No, it closes the double-quote started at `"schema `.